lib/routers/task/index.js

```diff
@@ -21,7 +22,7 @@ router.get('/:taskId', (req, res, next) => {
 });
 
 router.put('/:taskId/status', (req, res, next) => {
-  return req.workflow.task(req.taskId).status({ status: req.body.status, meta: req.body })
+  return req.workflow.task(req.taskId).status({ status: req.body.status, meta: omit(req.body, 'status') })
```

Comment:
I'd probably go for a `pick(req.body, 'comment')` to only send what we know we need.